spec/handlers/newflow/student_signup/signup_form_spec.rb

```diff
@@ -0,0 +1,234 @@
+require 'rails_helper'
+
+module Newflow
+  module StudentSignup
+    describe SignupForm, type: :handler do
+      context 'when success' do
+        before(:all) do
+          load('db/seeds.rb')
+        end
+
+        let(:subject) do
```

Comment:
this isnt really a subject, and can be too easily confused with rspec's subject DSL method.

maybe rename to:

```
       let(:handler_call) do
          described_class.call(params: params)
        end
```

Reply:
Fixed.